### PR TITLE
Execute securesign e2e test suite

### PR DESCRIPTION
**Pull request by bouskaJ**

### CODE REVIEW

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-images

File: ci-operator/config/securesign/secure-sign-operator/securesign-secure-sign-operator-main.yaml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 base_images:
+  buildah:
+    name: stable
+    namespace: buildah
+    tag: v1.35
```
Comment (JasonPowr) [inline ci-operator/config/securesign/secure-sign-operator/securesign-secure-sign-operator-main.yaml:5]: I don't think we need to create a new image here, as part of the ci process the operator is built and deployed already so it should be available to run the tests against.

Unless I'm missing something?

File: ci-operator/config/securesign/secure-sign-operator/securesign-secure-sign-operator-main.yaml [hunk 2]
```diff
@@ -75,13 +79,29 @@ tests:
...
+        IMAGE_NAME=ttl.sh/test-$RANDOM:1h
+        buildah pull alpine:latest
+        buildah tag alpine:latest $IMAGE_NAME
+        buildah push $IMAGE_NAME
+        echo $IMAGE_NAME >> ${SHARED_DIR}/image_name
+      from: buildah
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
```
Comment (JasonPowr) [inline ci-operator/config/securesign/secure-sign-operator/securesign-secure-sign-operator-main.yaml:95]: Same as above

File: ci-operator/config/securesign/secure-sign-operator/securesign-secure-sign-operator-main.yaml [hunk 3]
```diff
@@ -75,13 +79,29 @@ tests:
...
     - as: test
       cli: latest
       commands: |
         ./ci/openshift/tas-keycloak-install.sh
         set -x
+        export TEST_IMAGE=$(cat {SHARED_DIR}/image_name)
```
Comment (JasonPowr) [inline ci-operator/config/securesign/secure-sign-operator/securesign-secure-sign-operator-main.yaml:101]: Same as Above

**Review by JasonPowr — COMMENTED** [review]
@bouskaJ I've left some comments, just some things I can see

**Comment by bouskaJ** [comment]
Hello @sallyom I see you provided the initial configuration of this job so I guess, you will be able to help me. I am still getting 
```
  * could not run steps: step [input:operator-sdk] failed: failed to wait for importing imagestreamtags on ci-op-jx92m947/pipeline: timed out waiting for the condition
  * could not run steps: step [input:cli] failed: failed to wait for importing imagestreamtags on ci-op-jx92m947/pipeline: timed out waiting for the condition
  * could not run steps: step [input:root] failed: failed to wait for importing imagestreamtags on ci-op-jx92m947/pipeline: timed out waiting for the condition
  * could not run steps: step [input:buildah] failed: failed to wait for importing imagestreamtags on ci-op-jx92m947/pipeline: timed out waiting for the condition 
  * ```
Sorry to bother you with it. I am very new in the operator-ci...

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by bouskaJ** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by JasonPowr** [comment]
Seems as if you need to run `make ci-operator-config` you can also run `make update` which should catch anything missing https://github.com/openshift/release/blob/master/Makefile#L55

```
ERROR: This check enforces that CI Operator configuration YAML files are generated
ERROR: correctly. We have automation in place that updates these configs and
ERROR: new changes to these configurations should be followed with a re-generation.
ERROR: Run the following command to re-generate the CI Operator configurations:
ERROR: $ make ci-operator-config
ERROR: The following errors were found:
```

**Review by osmman — APPROVED** [review]

**Pull request merged**